tests/integration/targets/ini_file/tasks/tests/03-regressions.yml

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
a lot of integration tests could be called the same, because a lot are written to avoid regressions. Please consider to rename the file, `03-encoding.yml` or `03-charset.yml` for example.

Reply:
I renamed it to `03-encoding.yml`.